cpp/include/Ice/InputStream.h

```diff
@@ -591,7 +583,8 @@ namespace Ice
             }
         }
 
-#ifdef ICE_UNALIGNED // Optimization with unaligned reads
+#if defined(ICE_UNALIGNED) || (defined(_WIN32) && defined(ICE_API_EXPORTS))
```

Comment:
What about renaming this macro to `ICE_SUPPORT_UNALIGMENT_READ`

Reply:
I prefer a short / easier to type name. And it's not about supporting unaligned read, but about wanting to use them.